packages/trader/src/Stores/Modules/Trading/Helpers/proposal.ts

```diff
@@ -36,6 +36,25 @@ type TObjExpiry = {
     date_expiry?: number;
 };
 
+type TValidationParams =
```

Comment:
these don't come defined from deriv/api-types?

Reply:
Because in our current version of deriv/api-types they are absent. These fields were added by BE relatively not so long ago, so we will need a new version of deriv/api-types to make it work without my extra type